run.sh

```diff
@@ -6,8 +6,8 @@
 docker run\
   --env SSH_AUTH_SOCK=/ssh-agent\
   -v $SSH_AUTH_SOCK:/ssh-agent\
-  -v "$(pwd)/rosindex:/rosindex:rw"\
-  -v "$(pwd)/checkout:/checkout:rw"\
+  -v "$(pwd)/workdir:/workdir:rw"\
+  -v "$(pwd)/rosindex:/workdir/rosindex:rw"\
```

Comment:
Where you planning on keeping `rosindex/` as a submodule?

Reply:
I'd like to make this config independent too and remove it. But for now I was just leaving it in the same place.